pkg/rewards/claim.go

```diff
@@ -121,10 +123,15 @@ func Claim(cCtx *cli.Context, p utils.Prompter) error {
 		return eigenSdkUtils.WrapError("failed to fetch claim amounts for date", err)
 	}
 
+	claimableTokens, present := proofData.Distribution.GetTokensForEarner(config.EarnerAddress)
+	if !present {
+		return eigenSdkUtils.WrapError("no tokens claimable by earner", nil)
```

Comment:
nit: you can just do `errors.New` since you are anyway passing nil and there' no need to wrap errors.